lib/fog/azurerm/docs/storage.md

```diff
@@ -36,57 +63,104 @@ Check Storage Account Name Availability.This operation checks that account name
-        :location => 'West US',
-        :resource_group => '<Resource Group name>'
- )
+azure_storage_service.storage_accounts.create(
+  :name => '<Storage Account name>',
+  :location => 'West US',
+  :resource_group => '<Resource Group name>'
+)
 ```
 ## List storage accounts
 
 ##### List storage accounts in a subscription
 
 ```ruby
-    azure_storage_service.storage_accounts.each do |storage_acc|
-        puts "#{storage_acc.name}"
-        puts "#{storage_acc.location}"
-    end
+azure_storage_service.storage_accounts.each do |storage_acc|
+  puts "#{storage_acc.name}"
+  puts "#{storage_acc.location}"
+end
 ```
 ##### List storage accounts in a resource group
 
 ```ruby
-    storage_accounts  = azure_storage_service.storage_accounts(resource_group: '<Resource Group name>')
-    storage_accounts.each do |storage_acc|
-        puts "#{storage_acc.name}"
-        puts "#{storage_acc.location}"
-    end
+storage_accounts  = azure_storage_service.storage_accounts(resource_group: '<Resource Group name>')
+storage_accounts.each do |storage_acc|
+  puts "#{storage_acc.name}"
+  puts "#{storage_acc.location}"
+end
 ```
 
 ## Retrieve a single Storage Account
 
 Get a single record of Storage Account
 
 ```ruby
-      storage_acc = azure_storage_service
-                          .storage_accounts(resource_group: '<Resource Group name>')
-                          .get('<Storage Account name>')
-        puts "#{storage_acc.name}"
+storage_acc = azure_storage_service
+                .storage_accounts(resource_group: '<Resource Group name>')
+                .get('<Storage Account name>')
+puts "#{storage_acc.name}"
 ```
 
 ## Destroy a single Storage Account
 
 Get storage account object from the get method and then destroy that storage account.
 
 ```ruby
-      storage_acc.destroy
+storage_acc.destroy
 ```
 
 ## Delete a Disk
 
 Delete a Disk from a storage account. Disk must be in unlocked state i.e detached from server(virtual machine) to successfully perform this action.
 
 ```ruby
-      azure_storage_service.delete_disk('<Resource Group name>', '<Storage Account name>', '<Data Disk Name>')
+azure_storage_service.delete_disk('<Resource Group name>', '<Storage Account name>', '<Data Disk Name>')
+```
+
+## Create a storage container
+
+Create a storage container in the current storage account.
+
+```ruby
+container = azure_storage_service.create_container('<container name>')
+puts "#{container.name}"
+```
+
+## List storage containers
+
+List all the storage containers in the current storage accounts.
+
+```ruby
+containers = azure_storage_service.create_container('<container name>')
+containers.each do |container|
+  puts "#{container.name}"
+end
+```
+
+## Get storage container properties
+
+Get the storage container properties. The properties will not fetch the access control list. Call `get_container_access_control_list` to fetch it.
+
+```ruby
+properties = azure_storage_service.get_container_properties('<container name>')
```

Comment:
It is workable. But my question here is: 
```ruby
container = azure_storage_service.containers.get('<container name>')
properties = container.get_properties 
```
perform two requests to the server (the first one to list the containers and the second one to get the container properties) where only one `get_container_properties` API can make it.

I am OK to follow the convention, just have performance concern.